i_am_a_clinician_laboratory.html

```diff
@@ -205,6 +205,14 @@ <h3 class="padding-top-lg">Parameters Used for Matching and Score Output</h3>
     <td>&nbsp;</td>
     <td><span class='glyphicon glyphicon-ok'></span></td>
   </tr>
+  <tr>
+    <td><i>IRUD</i> (Japan)</td>
```

Comment:
This shouldn't be italics, so just remove `<i>` and `</i>` to leave `<td>IRUD (Japan)</td>`